.github/workflows/integration-tests.yml

```diff
@@ -19,26 +39,29 @@ jobs:
 
       - name: Clear cache
         run: |
-          rm -r ~/.triton/cache/
-        continue-on-error: true
+          rm -rf ~/.triton/cache/
 
       - name: Check imports
+        if: ${{ matrix.runner != 'macos-latest' }}
```

Comment:
Why do we skip this for macos?

Reply:
Thanks for your comment. Because `isort` is not available on `macos-latest`. I think this kind of check might not need Mac coverage as the correct format should be the same across machines; on contrary it is problematic if it gets different, like a newer version of clang-format returns different format.

If we shouldn't skip these, I will write an installation step for Mac,